src/webapi/web_socket.rs

```diff
@@ -72,10 +72,17 @@ pub struct WebSocket( Reference );
 
 impl IEventTarget for WebSocket {}
 
+/// The type of binary data being transmitted by the WebSocket connection.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket#Attributes)
 // https://html.spec.whatwg.org/#dom-websocket-binarytype
 #[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum BinaryType {
+    /// A Blob object represents a file-like object of immutable, raw data.
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Blob)
     Blob,
+    /// The ArrayBuffer object is used to represent a generic, fixed-length raw binary data buffer.
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer)
     ArrayBuffer
 }
```

Comment:
Can you rename it to `SocketBinaryType` and export it without the `web_socket` module? (Since we **don't** export other websocket related stuff in any namespace it'd be a little strange to make it only for this one single type.)

Reply:
Yes, will do !